src/main/webapp/WEB-INF/static/js/trialmanager/manageTrial.js

```diff
@@ -843,12 +843,13 @@ showAlertMessage,showMeasurementsPreview,createErrorNotification,errorMsgHeader,
 
 			$scope.printLabels = function () {
 				$uibModal.open({
-					template: '<dataset-option-modal modal-title="modalTitle" message="message"' +
+					template: '<dataset-option-modal modal-title="modalTitle" help-tool-url="helpToolUrl"  message="message"' +
 						' selected="selected" on-continue="forkPrintLabelFlows()"></dataset-option-modal>',
 					size: 'md',
 					controller: ['$scope', 'studyContext', function (scope, studyContext) {
 
 						scope.modalTitle = 'Create planting labels';
+						scope.helpToolUrl = 'https://bmspro.io/2193/training/bms-user-manual/label-design';
```

Comment:
look at [helpLinkService](https://github.com/IntegratedBreedingPlatform/Fieldbook/blob/3d7568bf6675820686084a256339c33672a682e1/src/main/webapp/WEB-INF/static/js/trialmanager/subobservations/subObservationTab.js#L14).
Alternatively, it's possible to inject the property using thymeleaf as a global variable, in Fieldbook/src/main/webapp/WEB-INF/pages/Common/includes/globals.html